rest-api/test/client_test/participant_test.py

```diff
@@ -52,12 +60,26 @@ def test_create_and_modify_participant(self):
       self.fail("Wrong If-Match header for update")
     except HttpException, ex:
       self.assertEqual(ex.code, 412)
-    response = self.client.request_json(
-          'Participant/{}'.format(participant_id), 'PUT', response,
+    new_response = self.client.request_json(
+          'Participant/{}'.format(participant_id), 'PUT', new_response,
           headers = { 'If-Match': last_etag})
-    self.assertEqual(response['biobankId'], biobank_id)
 
-    self.assertJsonEquals(response['providerLink'], [ provider_link_2])
+    self.assertEqual(new_response['biobankId'], biobank_id)
+    self.assertJsonEquals(new_response['providerLink'], [provider_link])
+    self.assertEqual(new_response['hpoId'], 'AZ_TUCSON')
+    self.assertEqual(new_response['awardee'], 'AZ_TUCSON')
+
+    new_response['providerLink'] = []
```

Comment:
Can you also add a test somewhere in here that shows that setting 'awardee' to 'UNSET' unpairs?

Reply:
done